CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 # UNRELEASED
 
+- feat: implement ChainGetTipSet in Lotus v2 APIs
```

Comment:
```suggestion
- feat: add experimental v2 APIs that are "F3 aware".  (TODO: expand this section significantly to cover where someone learns about the new APIs, how they enable them, and what expectations they should have around them - i.e., they may change)
```

I know the user documentation isn't the focus.  I'm thinking to leave a TODO so it's clear for whoever does the release that this feature needs to be expanded in the changelog (in case it hadn't gotten expanded earlier).